src/components/GuestCheckInForm.js

```diff
@@ -91,30 +87,27 @@ class CheckInForm extends React.Component {
               Room: {room.name}
             </option>
           ))}
-        </Select>
-        <TextField
+        </select>
+        <input
           placeholder="Name"
           className={
             this.state.errorName ? 'error hide-on-print' : 'hide-on-print'
           }
           onChange={event => this.setNameInput(event.target.value)}
           value={this.state.nameInput}
-          margin="normal"
         />
-        <div className="button-container">
-          {this.state.isCheckingIn ? (
-            <CircularProgress />
-          ) : (
-            <Button
-              className="hide-on-print"
-              variant="contained"
-              color="primary"
-              onClick={this.checkInGuest}
-            >
-              Check In
-            </Button>
-          )}
-        </div>
+        {this.state.isCheckingIn ? (
```

Comment:
I think this needs to be in a container so the UI doesn't jump around when switching between spinner and button